packages/loader/container-loader/src/container.ts

```diff
@@ -279,7 +279,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                         onClosed(err);
                     });
             }),
-            { start: true, end: true, cancel: "error" },
+            { start: true, end: true, cancel: "generic" },
```

Comment:
what layer logs this to error?

Reply:
Container.close() logs "ContainerClosed" error event in all workflows, when error happened during load, or if error happened after it. We can leave it as error, but looking at telemetry it's pretty much duplicates errors, and most container errors are actually on open. We can chose different design and not log "ContainerClosed" if container is still not loaded, not sure if I like it

Reply:
i'd prefer not to have redundant errors, but also don't want to lose errors, and the control flow of this function is over complicated, so just worried we could miss errors is not obvious ways if the method is modifed.

Reply:
I'm in favor of this, but yeah good point about future changes breaking assumptions. At least we will still have the data to follow-up if we suspect something is slipping through the cracks.

Speaking of which I did a quick join on `errorInstanceId`, and all load errors are showing up in ContainerClosed in recent days. Caveat there is that some errors aren't logging errorInstanceId, so it's a faulty experiment.  (Just fixed this in #8383)  But one we can repeat if we're ever concerned.